lib/experimental/Widgets/Layout/Dashboard/index.tsx

```diff
@@ -5,18 +5,31 @@ import {
   forwardRef,
   ReactNode,
   useEffect,
+  useMemo,
   useRef,
   useState,
 } from "react"
 import { Masonry } from "react-masonry"
 import { Widget } from "../../Widget"
 
 type DashboardProps = {
+  widgetWidth?: "sm" | "md" | "lg"
   children?: ReactNode[]
 }
 
 const _Dashboard = forwardRef<HTMLDivElement, DashboardProps>(
-  function Dashboard({ children }, ref) {
+  function Dashboard({ children, widgetWidth = "sm" }, ref) {
+    const maxWidgetWidth = useMemo(() => {
+      switch (widgetWidth) {
+        case "sm":
+          return 340
+        case "md":
+          return 480
+        case "lg":
+          return 640
```

Comment:
Another way to implement this so it's exhaustively checked is to define something like:

```tsx
const WidgetWidth = "sm" | "md" | "lg"

const widths: Record<WidgetWidth, number> = {
  "sm": 340,
  "md": 480,
  "lg": 640
}
```